test/functionalTest/README.md

```diff
@@ -0,0 +1,24 @@
+## How to run functional tests
+
+TBD
+
+## Known issues
+
+### Decimal numbers rounding
+
+You may find fails likes this one when you run the tests:
```

Comment:
fails => failures (or errors)

Reply:
Fixed in ee1d946dcc1ae74d5d1692a615919c32efccb625